src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
+    <PackageId>Microsoft.Azure.CognitiveServices.Language.SpellCheck</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
-    <VersionPrefix>1.2.0</VersionPrefix>
-    <AssemblyName>Microsoft.Azure.CognitiveServices.SpellCheck</AssemblyName>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Language.SpellCheck</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;SpellCheck API;REST HTTPclient;SpellCheckSDK;netcore451511</PackageTags>
-    <PackageReleaseNotes>This is a public release of the Cognitive Services SpellCheck SDK. Included with this release is support for SpellCheck API.</PackageReleaseNotes>
+    <PackageReleaseNotes>This is a preview release of the Cognitive Services SpellCheck SDK. Included with this release is support for SpellCheck API.</PackageReleaseNotes>
```

Comment:
If this is a preview, the package should be versioned as `2.0.0-preview`

Reply:
I updated the generate.cmd file in this directory. Does msbuild build.proj generate the files ? how I run this exactly ? where do i find msbuild and build.proj ?